### DISCUSSION TITLE
For the beginner....need help ( ITA or ENG )

### DISCUSSION THREAD

**Discussion by SC-Kello** [wlcrs/huawei_solar#496] [Connecting to the inverter]
Hallo , I got a Huawey SUN 2000 system with battery
I have install this integration over home assistant
settled the connection via Network
choose both port mentioned in the article , but no connection available
It is telling , Connection fault , with port ( 502 and 6607 )

Thanks from a beginner......

**Comment by JohnMcLear** [comment]
1.  What hostname are you trying to connect to?  The SDongle or SUN2000?
2.  What Slave IDs are you testing?  It's usually 1 but isn't always.  You can check this on the inverter using the Fusion Solar app.
3.  Did you enable RS485 over TCP in the Fusion Solar app?

**Comment by SC-Kello** [comment]
HiCould be that not understandI am trying to connect over network ( ip address ) no hostnameAbout slave , I have try with both optionI don’t know what need to soon point 3 , could you please show me the steps ?Sorry for my beginning steps……Il giorno 8 ott 2023, alle ore 11:42, John McLear ***@***.***> ha scritto:﻿

What hostname are you trying to connect to?  The SDongle or SUN2000?
What Slave IDs are you testing?  It's usually 1 but isn't always.  You can check this on the inverter using the Fusion Solar app.
Did you enable RS485 over TCP in the Fusion Solar app?

**Reply by JohnMcLear** [reply]
Try use hostname and ensure you can see that hostname from home assistant terminal using a ping command.

**Comment by SC-Kello** [comment]
I have try , below some screenshoot
￼
10.99.0.237 it is his good IP
When I try to configure…..see below

￼


￼

￼

With both port like website show ( 502 and 6607 )

The same if I use the IP address


Thanks

**Comment by JohnMcLear** [comment]
screenshots didn't work.

**Comment by SC-Kello** [comment]
Ok , host name and io address…..the same….Could be something over inverter that need to configure like interface or something like that ?The installer team no show me nothing , and I have end user login on the websiteIl giorno 8 ott 2023, alle ore 19:44, John McLear ***@***.***> ha scritto:﻿
screenshots didn't work.

**Comment by Roving-Ronin** [comment]
@SC-Kello 
To confirm that you have read the guide on how to install and configure, in the Wiki section ?
https://github.com/wlcrs/huawei_solar/wiki/Connecting-to-the-inverter

**Comment by SC-Kello** [comment]
Understood
Need to connect an adapter over inverter plug
I mean that no good for my expertise
No more possibility to connect without touch the inverter connectors ?


Thanks a lot

**Reply by Roving-Ronin** [reply]
@SC-Kello 

You have this?

https://www.italiatech.it/huawei-smart-dongle-wlan-fe-comunicazione-wifi-e-fast-ethernet-sdonglea-05.1.1.1414.gp.47297.uw

Use connect inverter to network using this. Then connect over network from HA, to the IP on home network of this.

**Comment by SC-Kello** [comment]
GreatIf I buy this , will be possible to capure fully data from the solar panel ( battery too ) with the application over HAIs it right ?Il giorno 9 ott 2023, alle ore 17:17, Roving Ronin ***@***.***> ha scritto:﻿
@SC-Kello
You have this?
https://www.italiatech.it/huawei-smart-dongle-wlan-fe-comunicazione-wifi-e-fast-ethernet-sdonglea-05.1.1.1414.gp.47297.uw
Use connect inverter to network using this. Then connect over network from HA, to the IP on home network of this.

**Reply by JohnMcLear** [reply]
Yes

**Comment by Pappnouse** [comment]
@SC-Kello I will summerize what I have understand. You have a Huawei Sun2000 with SDongle and LUNA2000 Battery, right? You can ping the IP of the SDonge, right?  Some addition questions from my site:
1. Is the inverter visible on Huawei FusionSolar (web site or mobile app) ?
2. Do you have enabled modbus communication on the inverter settings? Modbus TCP on the inverter is deactivated by default settings.
3. Do you have another modbus application which makes a connection to the inverter? Only one TCP Modbus connection is possible,
Best regards,
Stephan
